Typo fixes for LC solutions (#548)
moderndive/ModernDive_book: 94-appendixD.Rmd

```{r message=FALSE}
library(dplyr)
library(ggplot2)
library(nycflights23)
```

> **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`**  Repeat the above installing steps, but for the `dplyr`, `nycflights23`, and `knitr` packages. This will install the earlier mentioned `dplyr` package, the `nycflights23` package containing data on all domestic flights leaving a NYC airport in 2023, and the `knitr` package for writing reports in R. 

Author: ```suggestion
**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`**  Repeat the above installing steps, but for the `dplyr`, `nycflights13`, and `knitr` packages. This will install the earlier mentioned `dplyr` package, the `nycflights13` package containing data on all domestic flights leaving a NYC airport in 2013, and the `knitr` package for writing reports in R. 
```

**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** "Load" the `dplyr`, `nycflights23`, and `knitr` packages as well by repeating the above steps.

**Solution**: If the following code runs with no errors, you've succeeded!

```{r, eval=FALSE}
library(dplyr)

> library(nycflights23)

Author: ```suggestion
library(nycflights13)
```

```{r message=FALSE}
library(dplyr)
library(ggplot2)

> library(nycflights23)

Author: ```suggestion
library(nycflights13)
```

**Solution**: Because time is sequential: subsequent observations are closely related to each other.

> **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** Plot a time series of a variable other than `temp` for Newark Airport in the first 15 days of January 2023.

Author: ```suggestion
**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** Plot a time series of a variable other than `temp` for Newark Airport in the first 15 days of January 2013.
```

**Solution**: 

> * It would not work if we had a very large number of facets. For example, if we faceted by individual days rather than months, as we would have 365 facets to look at. When considering all days in 2023, it could be argued that we shouldn't care about day-to-day fluctuation in weather so much, but rather month-to-month fluctuations, allowing us to focus on seasonal trends.

Author: ```suggestion
* It would not work if we had a very large number of facets. For example, if we faceted by individual days rather than months, as we would have 365 facets to look at. When considering all days in 2013, it could be argued that we shouldn't care about day-to-day fluctuation in weather so much, but rather month-to-month fluctuations, allowing us to focus on seasonal trends.
```

**Solution**: See above.

**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** How many Envoy Air flights departed NYC in 2023?

> **Solution**: Envoy Air is carrier code `MQ` and thus 26397 flights departed NYC in 2023.

Author: ```suggestion
**Solution**: Envoy Air is carrier code `MQ` and thus 26397 flights departed NYC in 2013.
```

> **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What was the seventh highest airline in terms of departed flights from NYC in 2023? How could we better present the table to get this answer quickly?

Author: ```suggestion
**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What was the seventh highest airline in terms of departed flights from NYC in 2013? How could we better present the table to get this answer quickly?
```

see that the period in November, December, and January has the most variation in
weather, so you can expect very different temperatures on different days.

> **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What code would be required to get the mean and standard deviation temperature for each day in 2023 for NYC?

Author: ```suggestion
**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What code would be required to get the mean and standard deviation temperature for each day in 2013 for NYC?
```

**Solution**: Because `hour` is simply a value between 0 and 23; to identify a *specific* hour, we need to know which year, month, day and at which airport.

> **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What surprises you about the top 10 destinations from NYC in 2023?

Author: ```suggestion
**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** What surprises you about the top 10 destinations from NYC in 2013?
```

**Solution**: To narrow down the data frame, to make it easier to look at. Using `View()` for example.

> **`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** Create a new data frame that shows the top 5 airports with the largest arrival delays from NYC in 2023.

Author: ```suggestion
**`r paste0("(LC", chap, ".", (lc <- lc + 1), ")")`** Create a new data frame that shows the top 5 airports with the largest arrival delays from NYC in 2013.
```

library(ggplot2)
library(readr)
library(tidyr)

> library(nycflights23)

Author: ```suggestion
library(nycflights13)
```
